types/newman/index.d.ts

```diff
@@ -23,4 +170,12 @@ export interface NewmanRunExecutionAssertion {
 export interface NewmanRunExecutionAssertionError {
     message: string;
 }
-export function run(options: any, callback?: (err: Error | null, summary: NewmanRunSummary) => void): void;
+export interface NewmanRunFailure {
+    error: NewmanRunExecutionAssertionError;
+    /** The event where the failure occurred */
+    at: string;
+}
+export function run(
+    options: NewmanRunOptions,
+    callback?: (err: Error | null, summary: NewmanRunSummary) => void
```

Comment:
Are `Error` and `NewmanRunExecutionAssertionError` (supposed to be) related?

Reply:
They're different from what I could tell. This callback `Error` is for fatal newman errors but `NewmanRunExecutionAssertionError` is for test / script failures

callback error:
https://github.com/postmanlabs/newman/tree/release/3.x#newmanruncallbackerror-object--summary-object
> By default, only fatal errors, such as the ones caused by any fault inside Newman is passed on to this argument.

execution error:
https://github.com/postmanlabs/newman/blob/release/3.x/lib/run/summary.js#L240
seem to have errors that could be a syntax error, assertion error, or other error.

Reply:
Thanks for confirming!